core/domain/user_jobs_one_off_test.py

```diff
@@ -589,6 +598,58 @@ def test_weekly_stats_for_multiple_explorations(self):
             }
         }])
 
+    def test_stats_for_multiple_weeks(self):
```

Comment:
The length of the final weekly_stats is still 1. Shouldn't it be 2 ? (Since the one-off job had already been run once before, and then it was run for a further date!) ?